lib/registries/addon/components/registries-license-picker/component.ts

```diff
@@ -1,15 +1,39 @@
 import Component from '@ember/component';
+import { action } from '@ember/object';
 
+import { ChangesetDef } from 'ember-changeset/types';
 import { layout } from 'ember-osf-web/decorators/component';
-import { LicenseManager } from 'osf-components/components/editable-field/license-manager/component';
+import License from 'ember-osf-web/models/license';
+import { QueryHasManyResult } from 'ember-osf-web/models/osf-model';
+import Registration from 'ember-osf-web/models/registration';
 
 import styles from './styles';
 import template from './template';
 
+export interface LicenseManager {
+    onSave?: () => void;
+    onError?: () => void;
+    onCancel?: () => void;
+    onInput?: () => void;
+    changeLicense: (selected: License) => void;
+    registration?: Registration | ChangesetDef;
+    requiredFields: string[];
+    selectedLicense: License;
+    licensesAcceptable: QueryHasManyResult<License>;
+}
+
 @layout(template, styles)
 export default class RegistriesLicensePicker extends Component {
     manager!: LicenseManager;
 
+    shouldShowButtons: boolean = false;
+    inEditMode: boolean = true;
```

Comment:
So these aren't being modified in the component, but it's setting the default value in the case that it isn't an editable component, which passes through `inEditMode`.

Reply:
The `LicenseManager` exposes `inEditMode` as https://github.com/CenterForOpenScience/ember-osf-web/blob/d6f938a73a2406c261f59ddced347b0640ada271/lib/osf-components/addon/components/editable-field/license-manager/component.ts#L53
so you need to check it otherwise (on the overview page) you'd allow people to modify (or attempt to) the license without proper permissions.

Reply:
Ah, good point! Instead of doing this inside the component, IMHO, it should be conditionally invoked here:
https://github.com/CenterForOpenScience/ember-osf-web/blob/d56aeff71cb747ef74ce1921e48772d592ca1d1c/lib/registries/addon/components/registries-metadata/template.hbs#L200-L202